terraform/aws/analytical-platform-data-production/s3-glue-crawler/glue-catalogs.tf

```
@@ -1,3 +1,7 @@
 resource "aws_glue_catalog_database" "alpha_vcms_data" {
   name = "alpha-vcms-data"
 }
+
+resource "aws_glue_catalog_database" "alpha_vcms_data_2" {
+  name = "alpha_vcms_data"
+}
```

```suggestion
resource "aws_glue_catalog_database" "alpha_vcms_data" {
  name = "alpha_vcms_data"
}
```

Can the original not be replaced?

---

`alpha-vcms-data` is currently a source database in Create a Derived Table. I want to create `alpha_vcms_data` first, run the crawler, then update the source in CaDT (i.e. remove `alpha-vcms-data` and add `alpha_vcms_data`).

---

cool, when you are ready to retire `"alpha-vcms-data"`, get in touch and we can move resources around in the Terraform state